src/DoubleQueue.java

```diff
@@ -0,0 +1,88 @@
+public class DoubleQueue {
+    int [] array;
+    int size;
+    int lastQueueValue;
+
+    public int [] initializationQueue(int size) {
+        return array = new int[size];
+    }
+
+    public void addQueueTail(int [] array1, int value) {
```

Comment:
Переменные надо именовать так, чтобы они отражали. Иначе, это очень плохая практика.
```suggestion
    public void addQueueTail(int [] arr, int value) {
```